src/stan-users-guide/measurement-error.Rmd

```diff
@@ -285,7 +285,7 @@ transformed data {
   array[J] real<lower=0> sigma;
   for (j in 1:J) {
     y[j] = log(r_t[j]) - log(n_t[j] - r_t[j])
-            - (log(r_c[j]) - log(n_c[j] - r_c[j]);
+            - ((log(r_c[j]) - log(n_c[j] - r_c[j]));
```

Comment:
Shouldn't this only have a single parenthesis added, otherwise it's still unbalanced?

```suggestion
            - (log(r_c[j]) - log(n_c[j] - r_c[j]));
```

Reply:
ah yes 👍